templates/gig/gig_detail.html

```diff
@@ -256,18 +256,20 @@
                     {% trans "Comments" %}
                 </div></div>
             </div>
-            <div class="card-body">
+            <div class="card-body" id="gig_commentcard" hx-get="/gig/{{ gig.id }}/comments" hx-trigger="load">
                 <div class="row">
```

Comment:
All of the children are being replaced on load, so I think we can just get rid of them.

Reply:
Yes - this is leftover and I forgot to remove.